_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/menu/menu--topics-menu.html.twig

```diff
@@ -50,10 +50,10 @@
         {% endif %}
       </li>
     {% endfor %}
-      <li class="rhd-nav-all-topics"><a href="/blog/categories">All Topics<span class="icon"><i class="fas fa-caret-right"></i></span></a></li>
+      <li class="rhd-nav-all-topics"><a href="https://developers.stage.redhat.com/blog/categories/">All Topics<span class="icon"><i class="fas fa-caret-right"></i></span></a></li>
```

Comment:
Why are we changing this to staging, just to change it back to prod? Could we change this up to go to the drupal url and let the export fix the rest up?